src/drivers/mct/main/prep_ocn_mod.F90

```diff
@@ -48,6 +48,8 @@ module prep_ocn_mod
 
   public :: prep_ocn_get_x2oacc_ox
   public :: prep_ocn_get_x2oacc_ox_cnt
+  ! Sarat: Dummy variable added to workaround PGI compiler bug (PGI 17.9) as of Oct 23, 2017
+  public :: dummy_pgibugfix
```

Comment:
This is ugly - can we add 
#ifdef CPRPGI around it?

Reply:
@amametjanov , can you comment here?

Reply:
Looks like it's more specific than CPRPGI, was needed for v17.9  on the SummitDev machine: see https://github.com/ACME-Climate/ACME/pull/1908.

Reply:
We can at least limit it to PGI builds with that cppdef.

Reply:
There is a new `SUMMITDEV_PGI` macro in that PR, so `#ifdef SUMMITDEV_PGI` should be just the right scope.